scout/server/blueprints/variants/controllers.py

```diff
@@ -328,33 +339,19 @@ def compounds_need_updating(compounds, dismissed):
     return False
 
 
-def parse_variant(
-    store,
-    institute_obj,
-    case_obj,
-    variant_obj,
-    update=False,
-    genome_build="37",
-    get_compounds=True,
-    case_dismissed_vars=[],
-):
-    """Parse information about variants.
-    - Adds information about compounds
-    - Updates the information about compounds if necessary and 'update=True'
+def update_compounds(store, variant_obj, case_dismissed_vars):
```

Comment:
So nice with extracting the code fro parse_variant and creating new smaller functions! 🥇